src/Metadata/class-post-builder.php

```diff
@@ -433,7 +433,7 @@ private function get_coauthor_names( int $post_id ): array {
 		if ( class_exists( 'coauthors_plus' ) ) {
 			global $post, $post_ID, $coauthors_plus;
 
-			if ( ! ( $post_id > 0 ) && $post_ID ) {
+			if ( $post_id <= 0 && $post_ID ) {
```

Comment:
Besides the [code smell flagged by SonarCloud](https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&id=Parsely_wp-parsely&open=AYU5qVa6JSGsJPDrTNjA), I do not think that the extra parentheses are needed here.